src/test/objectstore/store_test.cc

```diff
@@ -6925,7 +6925,7 @@ TEST_P(StoreTestSpecificAUSize, ExcessiveFragmentation) {
     stringify((uint64_t)2048 * 1024 * 1024).c_str());
 
   ASSERT_EQ(g_conf().get_val<Option::size_t>("bluefs_alloc_size"),
-	    1024 * 1024U);
+	    0);
```

Comment:
AFAIR this check is intended to make sure bluefs_alloc_size is properly set for this test case to succeed. I.e. bluefs_alloc_size is expected to be 1MB. So wondering if it passes with 0? Perhaps needs explicit set to 1MB...

There is another test case which uses conf->bluefs_alloc_size, perhaps needs checking as well:
TEST_P(StoreTest, allocateBlueFSTest) {
...
uint64_t to_alloc = g_conf().get_val<Option::size_t>("bluefs_alloc_size");

Reply:
This change was made to address http://pulpito.ceph.com/nojha-2019-08-03_02:53:42-rados-wip-bluefs-alloc-size-distro-basic-smithi/4175755/, in the other test case we do not check for equality